packages/uui-color-picker/lib/uui-color-picker.element.ts

```diff
@@ -661,28 +661,29 @@ export class UUIColorPickerElement extends LabelMixin('label', LitElement) {
   }
 
   private _renderPreviewButton() {
-    return html`<button
-        type="button"
-        slot="trigger"
-        aria-label="${this.label || 'Open Color picker'}"
-        class=${classMap({
-          'color-picker__trigger': true,
-          'color-dropdown__trigger--disabled': this.disabled,
-          'color-dropdown__trigger--small': this.size === 'small',
-          'color-dropdown__trigger--medium': this.size === 'medium',
-          'color-dropdown__trigger--large': this.size === 'large',
-          'color-picker__transparent-bg': true,
-        })}
-        style=${styleMap({
-          '--preview-color': `hsla(${this.hue}deg, ${this.saturation}%, ${
-            this.lightness
-          }%, ${this.alpha / 100})`,
-        })}
-        ?disabled=${this.disabled}
-        @click=${this.openColorPicker}
-        aria-haspopup="true"
-        aria-expanded="false"></button>
       <uui-popover placement="bottom-start" @close=${this.closeColorPicker}>
```

Comment:
```suggestion
      return html`<uui-popover placement="bottom-start" @close=${this.closeColorPicker}>
```
Seems like there is a forgotten return. 😊

Reply:
Yes, some merge conflicts 😁